-scripts/generate-site.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+PLAYBOOK="$1"
+
+BUILD_DIR="build/site/tinymce"
+
+SOURCE_VERSION="latest"
+SOURCE_DIR="$BUILD_DIR/$SOURCE_VERSION/"
+
+DEST_VERSION=6
+DEST_DIR="$BUILD_DIR/$DEST_VERSION/"
+
+rm -rf "$BUILD_DIR"
```

Comment:
Do want to echo that the previous build dir is being removed?

Reply:
Good idea, I believe `antora playbook` would overwrite it anyway but this was ensuring both subdirectories were reset.